ChatRPG/ChatRPG/Pages/Dashboard.razor

```diff
@@ -0,0 +1,61 @@
+﻿@page "/Dashboard"
+@using ChatRPG.Data
+
+<h1>Dashboard</h1>
+
+<div class="conversation">
+    <div class="conversation-text">
+        @foreach (string message in _conversation)
+        {
+            <p>@message</p>
+        }
+    </div>
+</div>
+
+<div class="user-prompt">
+    <input type="text" @bind="_userInput" placeholder="What do you do?" @onkeyup="@EnterKeyHandler"/>
+    <button @onclick="SendPrompt">Send</button>
+</div>
+
+
+@code {
+    private readonly FileUtility _fileUtil = new FileUtility();
+    readonly List<string> _conversation = new List<string>();
+    string _userInput = "";
+
+    async Task EnterKeyHandler(KeyboardEventArgs e)
+    {
+        if (e.Code is "Enter" or "NumpadEnter")
+        {
+            await SendPrompt();
+        }
+    }
+
+    async Task SendPrompt()
+    {
+        if (!string.IsNullOrWhiteSpace(_userInput))
+        {
+            // Use the user input to generate a response
+            string userMessage = $"You: {_userInput}";
+            _conversation.Add(userMessage);
+            await _fileUtil.UpdateSaveFileAsync(userMessage);
+            string response = ProcessUserInput(_userInput);
+            string gameMessage = $"Game: {response}";
+            _conversation.Add(gameMessage);
+            await _fileUtil.UpdateSaveFileAsync(gameMessage);
```

Comment:
Det kunne være fint hvis der kun blev saved én gang frem for både efter userMessage og efter gameMessage

Reply:
Kan du bedre lide det nye?

Reply:
Yes, du kan også lave det til en record da den immutable, ikke har instance methods og skal indeholde begge beskeder (records kan kun laves med constructor)

Reply:
Very nice idea. Jeg har ikke brugt records før selv, så tænkte ikke over det. Det bruger vi bare i stedet.

Reply:
Alright, records are actually pretty hot, ngl

Reply:
Haha yeah 😎 resolver!